### DISCUSSION TITLE
在MOD管理界面添加下载mod的快捷方式

### DISCUSSION THREAD

**Discussion by HuiYangaliu** [Meloong-Git/PCL#2570] [提问] [answered]
下载整合包后想为整合包添加自己喜欢的mod，就得退出mod管理界面，然后点击下载 选择mod模块 搜索mod
所以我建议在mod管理界面添加快捷方式转到下载mod界面

**Comment by zkitefly** [comment, marked as answer]
~你可以发个 issue，不过我记得有重复的？（是吗？）~

https://github.com/Hex-Dragon/PCL2/issues/1336
